### DISCUSSION TITLE
PaddleSpeech的TTS模块有其他说话人类型吗？我找来找去只有一个女播音员类型的声音？

### DISCUSSION THREAD

**Discussion by zhijianli** [PaddlePaddle/PaddleSpeech#2004] [Q&A] [answered]
PaddleSpeech　的TTS模块有其他说话人类型吗？我找来找去只有一个女播音员类型的声音？

**Comment by yt605155624** [comment, marked as answer]
可以更换多说话人数据集
- 中文: aishell3
- 英文: vctk

通过 `spk_id` 选择不同的说话人

参考: 
- https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/demos/text_to_speech
- https://github.com/PaddlePaddle/PaddleSpeech/issues/1620
